types/openapi-factory/index.d.ts

```diff
@@ -15,9 +15,27 @@ declare namespace OpenApi {
         rawBody?: boolean;
     }
 
+    interface Authorizer {
+        jwt: string,
+        principalId: string
+    }
+
+    interface RequestContext {
+        authorizer: Authorizer
+    }
+
+    interface HttpRequest {
+        headers?: {[key: string]: any},
+        queryStringParameters?: {[key: string]: any},
+        pathParameters?: {[key: string]: any},
+        httpMethod?: string,
+        body?: (object | string);
```

Comment:
Should be `body?: ({} | string)` or preferably `body?: any` because it can be anything